src/SprykerEco/Zed/Computop/ComputopConfig.php

```diff
@@ -256,32 +261,58 @@ public function isRefundShipmentPriceEnabled()
     /**
      * @api
      *
-     * @return array
+     * @return array<string>
      */
-    public function getBeforeCaptureStatuses()
+    public function getBeforeInitStatuses()
     {
         return [
             $this->getOmsStatusNew(),
-            $this->getOmsStatusAuthorized(),
-            $this->getOmsStatusAuthorizationFailed(),
-            $this->getOmsStatusCancelled(),
         ];
     }
 
     /**
      * @api
      *
-     * @return array
+     * @return array<string>
      */
-    public function getBeforeRefundStatuses()
+    public function getBeforeAuthorizeStatuses()
```

Comment:
since this method is new, return type should be added to it

Reply:
Fixed